website/src/components/misc/link.tsx

```diff
@@ -1,26 +1,14 @@
 import { GatsbyLinkProps, Link as GatsbyLink } from "gatsby";
-import { OutboundLink } from "gatsby-plugin-google-analytics";
 import React, { FC } from "react";
 
 export const Link: FC<
   Pick<GatsbyLinkProps<unknown>, "download" | "to" | "onClick"> & {
     prefetch?: false;
   }
 > = ({ to, prefetch = true, ...rest }) => {
-  const internal = /^\/(?!\/)/.test(to);
-
-  return internal ? (
-    prefetch ? (
-      <GatsbyLink to={to} {...rest} />
-    ) : (
-      <a href={to} {...rest} />
-    )
+  return prefetch ? (
+    <GatsbyLink to={to} {...rest} />
   ) : (
-    <OutboundLink
-      href={to}
-      target="_blank"
```

Comment:
what about these 2 props that usually are there for security?

```
target="_blank"
rel="noopener noreferrer"
```

Also it affects any link as in companies:

Usage:
```
            <Link to="https://www.autoguru.com.au">
              <Company {...AutoguruLogoSvg} />
            </Link>
```

Emit:
```
<a href="https://www.autoguru.com.au" target="_blank" rel="noopener noreferrer"><svg viewBox="0 0 123 24"><use href="#autoguru"></use></svg></a>
```